### PR TITLE
Update skipped expenses in Fyle

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

File: apps/fyle/queue.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django_q.tasks import async_task
+
+
+def async_post_accounting_export_summary(org_id: str, workspace_id: int) -> None:
```
Comment (Shwetabhk) [inline apps/fyle/queue.py:4]: Add some comments to the function please along with description for function params

File: apps/fyle/tasks.py [hunk 2]
```diff
@@ -48,6 +52,39 @@ def create_expense_groups(workspace_id: int, fund_source: List[str], task_log: T
     return task_log
 
 
+def __get_updated_accounting_export_summary(expense_id: str, is_synced: bool) -> dict:
+    return {
+        'id': expense_id,
+        'state': 'SKIPPED',
```
Comment (Shwetabhk) [inline apps/fyle/tasks.py:58]: lets also pass state from Functions Param no, in case we want to use the same function for any other state but Skipped

Comment (ashwin1111) [reply apps/fyle/tasks.py:58]: yeah, that will be covered in the next PR

File: apps/fyle/tasks.py [hunk 3]
```diff
@@ -48,6 +52,39 @@ def create_expense_groups(workspace_id: int, fund_source: List[str], task_log: T
...
+    expense_to_be_updated = []
+    expenses_to_be_skipped = Expense.objects.filter(
+        final_query,
+        id__in=expenses_object_ids,
+        expensegroup__isnull=True,
+        org_id=workspace.fyle_org_id
+    )
+
+    for expense in expenses_to_be_skipped:
+        updated_accounting_export_summary = __get_updated_accounting_export_summary(expense.expense_id, False)
```
Comment (Shwetabhk) [inline apps/fyle/tasks.py:75]: Need not store it in a vairable, can directly call the function on line 80

File: apps/fyle/tasks.py [hunk 4]
```diff
@@ -48,6 +52,39 @@ def create_expense_groups(workspace_id: int, fund_source: List[str], task_log: T
...
+        expense_to_be_updated.append(
+            Expense(
+                id=expense.id,
+                is_skipped=True,
+                accounting_export_summary=updated_accounting_export_summary
```
Comment (Shwetabhk) [inline apps/fyle/tasks.py:80]:
```suggestion
                accounting_export_summary=__get_updated_accounting_export_summary(expense.expense_id, 'SKIPPED', False)
```

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**